project/src/main/webapp/script.js

```diff
@@ -0,0 +1,85 @@
+var map;
+var autocomplete;
+var photoURLS; 
+var image;
+
+function execute() {
+    initMap();
+    search();
+    getCatalog();
+}
+
+function initMap() {
+    map = new google.maps.Map(document.getElementById("map"), {
+        center: { lat: -33.866, lng: 151.196 },
+        zoom: 15
+    });
+}
+
+function search() {
+    // autocompleted suggestions of user searches
+    var input = document.getElementById("pac-input");
+    autocomplete = new google.maps.places.Autocomplete(input);
+    autocomplete.setFields(["place_id", "geometry"]);
+}
+
+function getCatalog() {
+    //returns photos associated with user search
+    autocomplete.addListener("place_changed", function() {
+        var place = autocomplete.getPlace();
+
+        if (!place.geometry) {
+            return;
+        }
+
+        var search = document.getElementById("place-id").innerText = place.place_id;
+
+        var request = {
+            placeId: place.place_id,
+            fields: ["photos"]
+        };
+
+        service = new google.maps.places.PlacesService(map);
+        service.getDetails(request, function callback(results, status) { // this request will return an array of photos as a result
+            if (status == google.maps.places.PlacesServiceStatus.OK) {
+                photos = results.photos;
```

Comment:
Isn't the catalogue suppose to be the food/menu, won't this just display pictures of the restaurant?

Reply:
yes some pictures will be of the place, trying to find a way to specify that

Reply:
it seems there isn't a way to specify food/menu items be shown with the places api, howon suggest we still use the images and add ratings and reviews